canonical-websites/www.ubuntu.com
# Remove bower version of global nav and associated bower config. Add NPM version of the global nav

## Done

- Remove bower global nav
- Remove bower config
- Add NPM global nav

## QA

- Check out this feature branch
- `./run clean`
- `./run`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- See that the global nav is present


## Issue / Card

Fixes #2242
